components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/Types.kt

```diff
@@ -31,6 +32,14 @@ internal fun mozilla.components.concept.sync.SyncAuthInfo.into(): SyncAuthInfo {
     )
 }
 
+/**
+ * Conversion from a generic [FrecencyThresholdOption] into its richer comrade within the 'places' lib.
```

Comment:
Not sure about the `richer` part, but I guess it could be! :) Hopefully they're maintained in lockstep though.

Reply:
Gonna keep it as-is since it's consistent with the VisitType `into()`.